code-challenges.test.js

```diff
@@ -48,6 +79,31 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+ 
+describe("nouns", () => {
+    it("takes in an array of words and returns an array with all the words capitalized")
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+    expect(nouns(randomNouns1)).toEqual(randomNouns1)
+    expect(nouns(randomNouns2)).toEqual(randomNouns1) 
+})
+
+const nouns = (randomNouns1, randomNouns2) => {
+    return randomNouns1.map(value => {
+        value.toUpperCase("s", "p", "t", "c", "n") } {
+            return randomNouns2.map(value =>  {
+                value.toUpperCase('t', 'd', 'c', 'm', 'd') 
+            })
+        }
+    
+    )}   
```

Comment:
I like that you thought of higher-order functions here!  This problem is a bit tricky since you will need to break apart the first letter from each word, run .toUpperCase, and then join it to the rest of the word again.